R/IOHexperimenter.R

```diff
@@ -1,68 +1,92 @@
+#' @importFrom assertthat assert_that
+#' @import Rcpp
+#' @useDynLib IOHexperimenter
+NULL
+
 #' S3 class 'IOHexperimenter'
 #'
-#' @param dims Numerical 
+#' @param dims Numerical
 #' @param functions Numerical
 #' @param instances Numerical Whether the underlying optimization algorithm performs a maximization?
+#' @param algorithm_info Additional information about the algorithm you plan on running
+#' @param algorithm_name The name of the algorithm you plan on running
+#' @param data.dir Where the data should be stored
+#' @param cdat Whether or not to generate a .cdat-file
+#' @param idat Integer
+#' @param tdat What frequency to use in a .tdat-file
+#' @param param.track Which parameters to track
 #'
 #' @return A S3 object 'DataSet'
 #' @export
 #'
-IOHexperimenter <- function(dims = c(100, 500, 1000, 2000, 3000), 
+IOHexperimenter <- function(dims = c(100, 500, 1000, 2000, 3000),
   functions = seq(23), instances = seq(100), algorithm_info = '', algorithm_name = '',
   data.dir = './data', cdat = FALSE, idat = 0, tdat = 3, param.track = '') {
-  
+
   assert_that(is.numeric(dims))
   assert_that(is.numeric(functions))
   assert_that(is.numeric(instances))
   base_evaluation_triggers <- 3
-  
+
   # intialize the backend C code
   c_init_suite(
-    paste0(functions, collapse = '-'), 
-    paste0(dims, collapse = '-'),
-    paste0(instances, collapse = '-')
+    paste0(functions, collapse = ','),
+    paste0(dims, collapse = ','),
+    paste0(instances, collapse = ',')
   )
-  
+
   # intialize the observer
   c_init_observer(
-    data.dir, algorithm_name, algorithm_info, 
+    data.dir, algorithm_name, algorithm_info,
```

Comment:
Not exactly for this line.... Please ask Furong to verify the default value for parameters like number_interval_triggers, ....

Reply:
I looked at the defaults with Furong, and they now match his original example configuration.

Reply:
Good! Thx!